refbox/src/tournament_manager/mod.rs

```diff
@@ -1867,6 +2104,14 @@ impl TournamentManager {
     }
 
     pub fn next_update_time(&self, now: Instant) -> Option<Instant> {
+        if self.time_pause_confirmation.is_some() {
+            if let Some(ref pause_conf) = self.time_pause_confirmation {
+                return Some(pause_conf.pause_began + pause_conf.duration_of_pause);
+            } else {
+                return None;
+            }
+        }
```

Comment:
The outer `if` is redundant here
```suggestion
        if let Some(ref pause_conf) = self.time_pause_confirmation {
            return Some(pause_conf.pause_began + pause_conf.duration_of_pause);
        } else {
            return None;
        }
```